dev-scripts/get.js

```diff
@@ -126,10 +133,10 @@ caporal
   .option('--regulars-only', 'Download only internals', caporal.BOOL, false)
   .option('--excludes <exclude>', 'Exclude a module', caporal.REPEATABLE, [])
   .option('--production', 'Leave only production files')
+  .option('--verbose, -v', 'Prints additional data on error', false)
```

Comment:
Can u explain to me what this does please

Reply:
This is a script to bring all the modules, which runs when you use the command `yarn get` add an option `--verbose` so if a module's download fail one can look at a more informative error.